plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/KafkaConnector.java

```diff
@@ -105,4 +108,10 @@ public final void shutdown()
     {
         lifeCycleManager.stop();
     }
+
+    @Override
+    public Set<ConnectorCapabilities> getCapabilities()
+    {
+        return immutableEnumSet(REPORT_BYTES_WRITTEN);
```

Comment:
We should have a test asserting that the declaration is true.
This should be generic and go to `BaseConnectorTest`. cc @wendigo @hashhar 

This is similar to `TestingConnectorBehavior` used there except here, we do have product level declaration, so we should not define a new `TestingConnectorBehavior`.

Reply:
Done